Tools/autotest/rover.py

```diff
@@ -303,6 +303,41 @@ def test_sprayer(self):
         if ex:
             raise ex
 
+    def drive_max_rcin(self, timeout=30):
+        """Test max RC inputs"""
+        self.context_push()
+        ex = None
+
+        try:
+            self.progress("Testing max RC inputs")
+            self.wait_mode("MANUAL")
+
+            self.wait_ready_to_arm()
+            self.arm_vehicle()
+
+            self.set_rc(3, 2000)
+            self.set_rc(1, 1000)
+
+            tstart = self.get_sim_time()
+            while self.get_sim_time() < tstart + timeout:
```

Comment:
```suggestion
            while self.get_sim_time_cached() - tstart < timeout:
```

We subtract time as a pattern.  It doesn't matter so much in Python, but it's really important in the ardupilot code, so we do maintain the pattern in our autotest suite.

Reply:
Fixed